coreclient/src/clients/mod.rs

```diff
@@ -405,8 +405,7 @@ impl CoreUser {
                     }
                 })
                 .collect::<Result<Vec<_>, _>>()?;
```

Comment:
`extend` takes an iterator, so there is no need to to `collect` into a `Vec` first.

Reply:
True, but I need to discharge the `Result` before I can use it with `extend`. In any case, this will go away in the next PR when I strip out all the multi-client bits.

Reply:
Ah, makes sense. I missed that. Thank you!